test/unit/io_managers_test.py

```diff
@@ -78,3 +79,50 @@ def test_foreign_key_failure(sqlite_io_manager_fixture):
         foreign_key="(artistid)",
         rowids=[1],
     )
+
+
+def test_extra_column_error(sqlite_io_manager_fixture):
```

Comment:
Is there an analogous `test_missing_column_error`?

Reply:
Yes! Just added one.